server/meeting.go

```diff
@@ -37,9 +37,10 @@ func (p *Plugin) GetMeeting(channelID string) (*Meeting, error) {
 		if err != nil {
 			return nil, err
 		}
+		paddedChannelName := strings.ReplaceAll(channel.Name, "-", "_")
 		meeting = &Meeting{
 			Schedule:      []time.Weekday{time.Thursday},
-			HashtagFormat: strings.Join([]string{fmt.Sprintf("%.15s", channel.Name), "{{ Jan02 }}"}, "-"),
+			HashtagFormat: strings.Join([]string{fmt.Sprintf("%.15s", paddedChannelName), "{{ Jan 2 }}"}, "_"),
```

Comment:
Why did the dash change to an underscore, and why did `Jan02` cange to `Jan 2`?

Reply:
Explained.

Reply:
Not sure why `Jan02` changed to `Jan 2` though

Reply:
Fixed it since most of our docs use Jan 2.